### PR TITLE
#497 Added 1.0.1 and 1.1.0 as valid versions

### PR DESCRIPTION
**Pull request by jmax01**
I simply cloned 1.0.0 code for both.
This may not be sufficient.

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by jmax01** [comment]
I simply cloned 1.0.0 code for both.
This may not be sufficient.

**Comment by manjeethike** [comment]
We need to upgrade to Kafka 1.1.0, We use Kafka Manager for it. Please merge this PR so that we can deploy latest build with Kafka 1.1.0 Support

**Comment by danielmotaleite** [comment]
So the CLA is signed, what stop this from being merged?

**Comment by maxtuzz** [comment]
Can this be merged? 

Many of us are resorting to running our own kafka managers in production with this feature included because it's taking much too long to be added as an official release.

File: app/kafka/manager/utils/LogkafkaNewConfigs.scala [hunk 1]
```diff
@@ -32,7 +32,9 @@ object LogkafkaNewConfigs {
     Kafka_0_10_2_1 -> logkafka82.LogConfig,
     Kafka_0_11_0_0 -> logkafka82.LogConfig,
     Kafka_0_11_0_2 -> logkafka82.LogConfig,
-    Kafka_1_0_0 -> logkafka82.LogConfig
+    Kafka_1_0_0 -> logkafka82.LogConfig,
```
Comment (sm4rk0) [inline app/kafka/manager/utils/LogkafkaNewConfigs.scala]: Please have a look at [this comment](https://github.com/yahoo/kafka-manager/pull/510#issuecomment-395349386).

Comment (jmax01) [reply app/kafka/manager/utils/LogkafkaNewConfigs.scala]: I will update

File: test/kafka/manager/model/KafkaVersionTest.scala [hunk 2]
```diff
@@ -61,7 +63,9 @@ class KafkaVersionTest extends FunSuite {
       ("0.10.2.1","0.10.2.1"),
       ("0.11.0.0","0.11.0.0"),
       ("0.11.0.2","0.11.0.2"),
-      ("1.0.0","1.0.0")
+      ("1.0.0","1.0.0"),
+      ("1.0.1","1.0.1"),
+	  ("1.1.0","1.1.0")
```
Comment (robsonpeixoto) [inline test/kafka/manager/model/KafkaVersionTest.scala]: code style?

File: app/controllers/Logkafka.scala [hunk 3]
```diff
@@ -137,6 +141,8 @@ class Logkafka (val messagesApi: MessagesApi, val kafkaManagerContext: KafkaMana
           case Kafka_0_11_0_0 => (defaultCreateForm.fill(kafka_0_11_0_0_Default), clusterContext)
           case Kafka_0_11_0_2 => (defaultCreateForm.fill(kafka_0_11_0_2_Default), clusterContext)
           case Kafka_1_0_0 => (defaultCreateForm.fill(kafka_1_0_0_Default), clusterContext)
+		  case Kafka_1_0_1 => (defaultCreateForm.fill(kafka_1_0_1_Default), clusterContext)
```
Comment (robsonpeixoto) [inline app/controllers/Logkafka.scala]: Use only spaces. There are tabs here.

**Comment by kovalskyi** [comment]
Is there any progress on this one? @jmax01 could you update your pull request please? Or perhaps I could create a separate pull request with requested changes?

**Comment by jmax01** [comment]
I should be able to get to it this week, possibly tomorrow.

**Comment by pichouk** [comment]
Since #519 is merged, I guess this PR can be closed ?

**Comment by Ashish123gs** [comment]
can we add support for 1.1.1 please ?

**Comment by danielmotaleite** [comment]
well, 2.1.0 was released some days ago! :D

**Comment by Ashish123gs** [comment]
yeah , but we are still at 1.1.1 and might take time to move to 2.x series :)

**Pull request closed without merging**